addons/pos_loyalty/static/src/tours/PosLoyaltyLoyaltyProgramTour.js

```diff
@@ -57,8 +57,8 @@ ProductScreen.do.clickDisplayedProduct('Whiteboard Pen');
 ProductScreen.check.selectedOrderlineHas('Whiteboard Pen', '2.00');
 ProductScreen.do.clickDisplayedProduct('Whiteboard Pen');
 ProductScreen.check.selectedOrderlineHas('Whiteboard Pen', '3.00');
-ProductScreen.do.clickDisplayedProduct('Whiteboard Pen');
 PosLoyalty.check.isRewardButtonHighlighted(false);
+ProductScreen.do.clickDisplayedProduct('Whiteboard Pen');
```

Comment:
@caburj The test was working because the `isRewardButtonHighlighted` was not working correctly. I think this change make sense. Does it look ok to you ?